site/layouts/shortcodes/example.html

```diff
@@ -32,10 +32,10 @@
         <small class="font-monospace text-body-secondary text-uppercase">{{- $lang -}}</small>
         <div class="d-flex ms-auto">
           <button type="button" class="btn-edit text-nowrap"{{ with $stackblitz_add_js }} data-sb-js-snippet="{{ $stackblitz_add_js }}"{{ end }} title="Try it on StackBlitz">
-            <svg class="bi" aria-hidden="true"><use xlink:href="#lightning-charge-fill"/></svg>
+            <svg class="bi" aria-hidden="true" focusable="false"><use xlink:href="#lightning-charge-fill"/></svg>
```

Comment:
Was needed for IE but we are no longer supporting IE.

Reply:
I did it for the consistency of the doc + I think it's fine having it atm since Bootstrap have it too. Feel free to open a PR on Bootstrap side if you want to remove all of them.

Reply:
And what about adding `focusable="false"` to bootstrap directly? 🤔 
https://github.com/twbs/bootstrap/blob/main/site/layouts/shortcodes/example.html

Reply:
I'll propose a PR on this subject on Bs directly!

Reply:
Great. I will resolve this comment once this change will be validated by their core team.